nix/packages.nix

```diff
@@ -1,7 +1,7 @@
 # SPDX-FileCopyrightText: 2023 Technology Innovation Institute (TII)
 #
 # SPDX-License-Identifier: Apache-2.0
-{inputs, ...}: {
+_: {
```

Comment:
I don't think you need to use `_: { }` here, it can be just `{ }` without any imports

Reply:
Fixed on https://github.com/tiiuae/sbomnix/pull/131/commits/055942644f43126441bb19133458a6a594b52e8c